recipes/default.rb

```diff
@@ -20,8 +20,22 @@
 # prime the search to avoid 2 masters
 node.save unless Chef::Config[:solo]
 
-package 'drbd8-utils' do
-  action :install
+case node['platform']
```

Comment:
Going by the support from DRBD, this should only support RHEL, Fedora, Suse, Debian, and Ubuntu.

I have only tested this on Ubuntu 14.04 and CentOS 6.5, but it should work on all centos 6.x 

http://www.drbd.org/en/doc/users-guide-83/ch-install-packages

Perhaps I should modify the Metadata file to reference only supported OSes?

-Haus